scripts/tls-certificate.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# Install certbot
+apt-get update
+apt-get install -y certbot acl
```

Comment:
you don't need to install acl.
You can achieve the same goal with `chmod o=rX`

Reply:
Yes, but I don't know the impact of opening fully.
For SSH key for exemple, it cannot work if it's world readable.

Reply:
It try it and it works for projector but you're right I don't the impact for other usage